proposals/1.0.0/packedMod.md

```diff
@@ -0,0 +1,80 @@
+Standard for packed CrossCode mods. This should simplify sharing and installation of mods.
+
+# Current Situation / Problems
+
+Mods have to be unzip by the user and installed my manually moving them into the mods folder. "Packed mods" are not standartized (rar, 7zip, tar.gz, ...). Not every OS supports every format.
+
+# Solution
+
+Standard for packed mods
+
+* zip of a mod with the extension `.ccmod` (<https://www.iana.org/assignments/media-types/application/zip>)
+* requires package.json in the main directory (see unpacked mods)
+
+## Advantage
+
+* Simplified installation (file association)
+* Standardised container
+* Easy purpose recognition after downloads (`What was this for?`)
+
+## Disadvantage
+
+* More dependencies (if the optional part is implemented)
+* System/OS dependend parts (if the optional part is implemented)
+* Difficulties in testing increase (if the optional part is implemented)
+* Higher programming costs
+
+# Requirement
+
+* Library to unzip the `ccmod` file (e.g. `zlib` - <https://nodejs.org/api/zlib.html>)
+* fs (read `package.json`)
+* fs (move folder)
+
+## Optional
+
+* Library to access windows regestry (e.g. `winreg` - <https://www.npmjs.com/package/winreg>)
+* Tool packed as .app (macOS specific)
+* fs (file edit)
+
+# File handling
```

Comment:
This title is a little unclear. Maybe `Installation methods` would be more clear